tests/draft4/items.json

```diff
@@ -19,6 +19,14 @@
                 "description": "ignores non-arrays",
                 "data": {"foo" : "bar"},
                 "valid": true
+            },
+            {
+                "description": "not array is valid",
```

Comment:
This one is redundant with "ignores non-arrays" above it for what I can tell, unless you're testing something different here, but not seeing it?